### DISCUSSION TITLE
ARGOCD UI is not showing appropriate app structure

### DISCUSSION THREAD

**Discussion by AnubhavSabarwal** [argoproj/argo-helm#2554] [General]
<img width="1761" alt="Screenshot 2024-02-27 at 10 01 09 PM" src="https://github.com/argoproj/argo-helm/assets/133980040/9d0bc300-37f7-4bea-a323-d9471e3de20f">

Earlier, it used to be 3 apps in one row and now, they are segregated, any workaround for this

**Comment by jmeridth** [comment]
@AnubhavSabarwal this would need to be filed at https://github.com/argoproj/argo-cd/issues (the upstream argo-cd project).  This project is only for the helm charts.
